core/field_checkbox.js

```diff
@@ -108,7 +108,7 @@ FieldCheckbox.prototype.configure_ = function(config) {
 FieldCheckbox.prototype.initView = function() {
   FieldCheckbox.superClass_.initView.call(this);
 
-  Blockly.utils.dom.addClass(
+  addClass(
```

Comment:
Question: might `dom.addClass` be more readable here?

Reply:
Done - I got some feedback in the opposite direction on a review yesterday, but I do sort of prefer the additional context.